test/http-api/interface/bootstrap.js

```diff
@@ -5,7 +5,10 @@ const test = require('interface-ipfs-core')
 const parallel = require('async/parallel')
 
 const DaemonFactory = require('ipfsd-ctl')
-const df = DaemonFactory.create({ exec: 'src/cli/bin.js' })
+const df = DaemonFactory.create({
+  exec: 'src/cli/bin.js',
+  port: 0
```

Comment:
No need to setup the port here. The port is just used when we want to use it as a client/server for Browser testing.